src/files/domain/models/FilePaginationInfo.ts

```diff
@@ -0,0 +1,52 @@
+export class FilePaginationInfo {
+  constructor(
+    public readonly page: number = 1,
+    public readonly pageSize: number = 10,
+    public readonly total: number = 0
```

Comment:
By reading this model without much context I don't understand what this "total" property refers to and why it can be 0. Maybe renaming it can help?

Reply:
I see 🤔 This total refers to the total number of files, the total number of elements to be paginated, maybe rename it to totalFiles? filesTotal?

The total number of pages is called here totalPages so I guess we can call this totalFiles

Reply:
totalFiles makes sense!